packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx

```diff
@@ -201,13 +201,14 @@ export const UploadControls = (): JSX.Element => {
   const [{ history, path }] = useControl({ type: 'NAVIGATE' });
   const [files, setFiles] = React.useState<File[]>([]);
   const [fileSelect, handleSelect] = useFileSelect(setFiles);
-
+  const [overwrite, setOverwrite] = React.useState(false);
```

Comment:
Maybe add a comment that overwrite is disabled by default?

Reply:
Added a comment in latest:
```
// preventOverwrite is enabled by default in our call to uploadData
// so we set overwrite to default to false to match in our UI
```